### PR TITLE
changed styling in button mark as present and mark as optin/optout

### PR DESCRIPTION
**Pull request by priyalpepcus**

### CODE REVIEW

File: src/dataGrid.css [hunk 1]
```diff
@@ -1995,8 +2030,14 @@ body,
         display: none;
     }
     .id-card-wrapper .buttonContainer:nth-child(4) {
-        display: block;
```
Comment (avi-jain-pepcus) [inline src/dataGrid.css]: remove specific target CSS from file.

File: src/dataGrid.css [hunk 2]
```diff
@@ -569,7 +569,7 @@ html, body {
     padding: 8px 16px;
 }
 .button-modal {
-    margin-left: 16px;
+    /*margin-left: 16px;*/
```
[the remark this answers is not in the document]
Comment (priyalpepcus) [reply src/dataGrid.css]: Removed.

File: src/dataGrid.css (hunk 1 above)
Comment (priyalpepcus) [reply src/dataGrid.css]: Removed and replaced with class name named as **button-container-mobile**.

**Comment by pepcus-pratik** [comment]
@priyalpepcus , rebase this branch

**Pull request merged**